src/js/stores/MesosStateStore.js

```diff
@@ -255,17 +255,21 @@ class MesosStateStore extends GetSetBaseStore {
     // the scheduler tasks or a list of Marathon application tasks.
     return frameworks.reduce(function (serviceTasks, framework) {
       let {tasks = [], completed_tasks = {}, name} = framework;
+
+      let allTasks = tasks.concat(completed_tasks).map(function (task) {
+        task.scheduler = name;
```

Comment:
We should really do this in the action when we fetch this data, if we need this information.
Tasks should already have scheduler ids. We should use that in stead.